astropy/io/fits/scripts/fitsinfo.py

```diff
@@ -0,0 +1,64 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+``fitsinfo`` is a command-line script based on astropy.io.fits for
+printing a summary of the HDUs in one or more FITS files(s) to the
+standard output.
+
+Example usage of ``fitsinfo``:
+
+1. Print a summary of the HDUs in a FITS file::
+
+    $ fitsinfo filename.fits
+
+    Filename: filename.fits
+    No.    Name         Type      Cards   Dimensions   Format
+    0    PRIMARY     PrimaryHDU     138   ()
+    1    SCI         ImageHDU        61   (800, 800)   int16
+    2    SCI         ImageHDU        61   (800, 800)   int16
+    3    SCI         ImageHDU        61   (800, 800)   int16
+    4    SCI         ImageHDU        61   (800, 800)   int16
+
+2. Print a summary of HDUs of all the FITS files in the current directory::
+
+    $ fitsinfo *.fits
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import argparse
+import astropy.io.fits as fits
+from astropy import log
+
+
+def fitsinfo(filename):
+    """
+    Print a summary of the HDUs in a FITS file.
+
+    Parameters
+    ----------
+    filename : str
+        The path to a FITS file.
+    """
+
+    try:
+        hdulist = fits.open(filename)
+        hdulist.info()
+        hdulist.close()
+    except IOError as e:
+        log.error(str(e))
```

Comment:
@embray The lowest log level setting in astropy is apparently 'ERROR':  http://docs.astropy.org/en/latest/logging.html.

Are you suggesting that I use `log.critical()` intstead of `log.error()`?  Because the lowest astropy log level is ERROR, they both will always be printed.  What's the distinction between error and critical?

Or would you prefer to take out the logging and simply let it raise the IOError exception with the traceback?

Note that `fitsheader` (e.g. https://github.com/astropy/astropy/blob/master/astropy/io/fits/scripts/fitsheader.py#L247)
uses `log.error` the same as I've done here.